spec/models/topic_tracking_state_spec.rb

```diff
@@ -594,6 +594,7 @@
     end
 
     it "includes tags when SiteSetting.navigation_menu is not legacy" do
+      SiteSetting.navigation_menu = "legacy"
```

Comment:
The description and the site setting here doesn't really tally now.

Reply:
Nothing has actually changed though. The actual test is a few lines down when we explicitly set the navigation menu to "sidebar" and we check for tags. The original test just assumed that the sidebar was off by default when setting up the test.

If the description needs to be updated, could you suggest a better one?

Reply:
O icic. The test is testing both `navigation_menu` == "legacy" and `navigation_menu` == "sidebar". I couldn't tell from just looking at the diff. All good here :+1: